gradle.properties

```diff
@@ -23,4 +23,5 @@ android.enableJetifier=false
 kotlin.code.style=official
 # Needed to enable Android data binding.
 android.databinding.enableV2=true
-
+# See: https://stackoverflow.com/q/70878643/3689782.
```

Comment:
NB: This interestingly seems to only matter when upgrading to Moshi 1.14 (which ended up not happening in this PR), so I may revert this.

Reply:
Revert this.

Reply:
Done.